doc/example/basic.txt

```diff
@@ -151,12 +151,7 @@ By default tox uses `pip`_ to install packages, both the
 package-under-test and any dependencies you specify in ``tox.ini``.
 You can fully customize tox's install-command through the
 testenv-specific :confval:`install_command=ARGV` setting.
-For instance, to use ``easy_install`` instead of `pip`_::
-
-    [testenv]
-    install_command = easy_install {opts} {packages}
-
```

Comment:
As for pip with different paramters, there's example just below the removed part.
``` 
    [testenv]
    install_command = pip install --pre --find-links http://packages.example.com --no-index {opts} {packages}```